bot/cogs/memes_cog.py

```diff
@@ -12,6 +12,7 @@
 
 class MemesCog(commands.Cog):
 
+    max_waldo_grid_size = 100
```

Comment:
make this a module constant, so it goes above the class def and below the logger